### PR TITLE
Add build aarch64 rpm inside ci.yml

### PR DESCRIPTION
**Pull request by matinlotfali**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -25,3 +28,48 @@ jobs:
...
+    - uses: actions/checkout@v3
+      with:
+        path: gstreamer1.0-drpai
+
+    - name: Replace the RZV2L gstreamer1.0-drpai with the current one.
+      run: |
+        rm -rf rzv2l/Build/meta-mistysom/recipes-drpai/gstreamer1.0-drpai
+        cp -r gstreamer1.0-drpai rzv2l/Build/meta-mistysom/recipes-drpai/
+        echo "SRC_URI = \"file://\${THISDIR}\"" >> rzv2l/Build/meta-mistysom/recipes-drpai/gstreamer1.0-drpai/bitbake-recipe.bb
+        echo "S = \"\${WORKDIR}/\${THISDIR}\"" >> rzv2l/Build/meta-mistysom/recipes-drpai/gstreamer1.0-drpai/bitbake-recipe.bb
```
Comment (mistyron) [inline .github/workflows/ci.yml:54]: Why do you write the strings to `bitbake-recipe.bb` just to rename it afterwards to `gstreamer1.0-drpai.bb` ?

Comment (matinlotfali) [reply .github/workflows/ci.yml:54]: I can rename it first and then write strings. Does the order matter?

Comment (mistyron) [reply .github/workflows/ci.yml:54]: no, but why aren't you echo-ing the strings into `gstreamer1.0-drpai.bb` directly? it looks like you're echo-ing into `rzv2l/Build/meta-mistysom/recipes-drpai/gstreamer1.0-drpai/bitbake-recipe.bb` jus to rename it afterwards to `rzv2l/Build/meta-mistysom/recipes-drpai/gstreamer1.0-drpai/gstreamer1.0-drpai.bb`. so why not echo to  `gstreamer1.0-drpai.bb` to begin with? I feel like I'm missing something, but what is it??? 🤔

Comment (matinlotfali) [reply .github/workflows/ci.yml:54]: `bitbake-recipe.bb` has its own contents. I am echoing to the end of it so it would override variables.

Comment (mistyron) [reply .github/workflows/ci.yml:54]: Yes, I see but it still doesn't make sense to me to do it this way...

Comment (matinlotfali) [reply .github/workflows/ci.yml:54]: @mistyron 
How would you suggest this to be implemented?

Comment (mistyron) [reply .github/workflows/ci.yml:54]: Haha, I can't believe that we're still discussing this. Nevermind, I see now what I've missed before.

**Pull request merged**